codegen/Cargo.toml

```diff
@@ -40,3 +40,10 @@ bitvec = { workspace = true }
 scale-info = { workspace = true, features = ["bit-vec"] }
 pretty_assertions = { workspace = true }
 frame-metadata = { workspace = true }
+
+[package.metadata.docs.rs]
+all-features = true
```

Comment:
Can we test this somehow? would `all-features = true` enable eg both `native` _and_ `web` and cause a compile error for instance, or does it avoid that?

Reply:
yeah, you are right I have to remove that